openfecwebapp/api_caller.py

```diff
@@ -9,11 +9,16 @@
 MAX_FINANCIALS_COUNT = 4
 
 
-def _call_api(path, filters):
+def _call_api(*path_parts, **filters):
     if api_key:
         filters['api_key'] = api_key
-    path = os.path.join(api_version, path.strip('/'))
+
+    path = os.path.join(api_version, *[x.strip('/') for x in path_parts])
     url = parse.urljoin(api_location, path)
+
+    print(url)
```

Comment:
Prefer to remove print statements before merge. I think using the `logging` module could be helpful here. That way we can write something like `logger.debug(url)`, and set the log level to `DEBUG` for development, and maybe `WARN` in production.